### PR TITLE
Build with esbuild

### PR DESCRIPTION
**Pull request by djhi**
I haven't noticed any difference in the build output for the demo. I checked its bundle analysis and tree shaking seems to work fine

### CODE REVIEW

File: packages/ra-input-rich-text/src/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './RichTextInput';
```
Comment (djhi) [inline packages/ra-input-rich-text/src/index.ts:1]: Breaking change as we now have a named export. However, we plan to replace this input with the tiptap version

**Review by WiXSL — APPROVED** [review]
I'll try it and builds everything without problem, even though I don't see an important improvement in time

**Review by fzaninotto — APPROVED** [review]

**Comment by fzaninotto** [comment]
That's awesome, but I find the build output too verbose:

image

Could you make it shorter?

**Comment by djhi** [comment]
Only completely silent

**Comment by fzaninotto** [comment]
sounds good to me

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**